evap/evaluation/views.py

```diff
@@ -94,3 +102,27 @@ def faq(request):
 
 def legal_notice(request):
     return render(request, "legal_notice.html", dict())
+
+
+@require_POST
+@login_required
+def feedback_send(request):
+    sender_email = request.POST.get("sender_email")
```

Comment:
please use the email of the logged in user instead.

@janno42 actually, do we want this? it wouldn't be anonymous then anymore... maybe we should at least put a disclaimer on top. and from a technical POV, does owa (still) allow sending from arbitrary email addresses?